src/App.vue

```diff
@@ -87,52 +87,54 @@ export default {
         this.name = "";
         this.surname = "";
         this.age2 = "";
-      } else if(!this.name || !this.surname || !this.age2){
-        alert("You need to fill in all inputs!")
+      } else {
+        alert("You need to fill in all inputs!");
       }
     },
     deleteInformation(index) {
       if (confirm("Do you want to delete this?")) {
-        this.information.splice(index);
+        this.information = this.information.filter((el, i) => i != index);
       }
     },
-    startEditing(index) {
-      this.information[index].editing = true;
-      this.information[index].edited.firstName =
-        this.information[index].firstName;
-      this.information[index].edited.lastName =
-        this.information[index].lastName;
-      this.information[index].edited.age = this.information[index].age;
+    startEditing(el) {
+      el.editing = true;
+      el.edited.firstName = el.firstName;
+      el.edited.lastName = el.lastName;
+      el.edited.age = el.age;
     },
-    saveData(index) {
-      const editedData = this.information[index];
-      if(!editedData.edited.firstName || !editedData.edited.lastName || !editedData.edited.age){
-        alert("You need to fill in all inputs!")
+    saveData(el) {
+      const editedData = el;
+      if (
+        !editedData.edited.firstName ||
+        !editedData.edited.lastName ||
+        !editedData.edited.age
+      ) {
+        alert("You need to fill in all inputs!");
         return;
       }
       editedData.firstName = editedData.edited.firstName;
       editedData.lastName = editedData.edited.lastName;
       editedData.age = editedData.edited.age;
-      editedData.fullName = ", " + editedData.edited.firstName + " " + editedData.edited.lastName;
+      editedData.fullName =
+        ", " + editedData.edited.firstName + " " + editedData.edited.lastName;
```

Comment:
Koristi template string ovde (`` sa ${} unutra).
Nikad ne koristi zarez u full name? Cemu to? Full name se sastoji od imena i prezimena, nema zareza.
U ovom delu gde koristis to
{{ el.firstName }} {{ el.lastName }} {{ el.fullName }} {{ el.age }}
Napravi funkciju koja ce da prihvata **el** kao parametar a da **vraca** vrednost koja ti treba sa zarezima.
Osatlo je sve top!!!